starport/templates/module/create/ibc.go

```diff
@@ -7,7 +7,6 @@ import (
 	"github.com/gobuffalo/genny"
 	"github.com/gobuffalo/plush"
 	"github.com/gobuffalo/plushgen"
-
```

Comment:
Several separators (i.e. blank lines) were removed, why? What's the justification? I'd advise reverting unnecessary changes in order to reduce the diff's size.